openrtx/include/core/state.h

```diff
@@ -35,32 +35,31 @@
  */
 typedef struct
 {
```

Comment:
Just another example: removing the extra spaces broke the alignment of the fields, making the code less clear